### ISSUE TITLE
How to use ace/tern with ace-builds?

### ISSUE THREAD

**Issue by jparismorgan** [ajaxorg/ace#4729]
Hi there. I am trying to use `tern` with ace-builds, but it is not working, as I'm getting errors when doing:
```
ace.config.loadModule('ace/ext/language_tools', () => {
      ace.config.loadModule('ace/ext/tern', () => {
            ...
```
<img width="797" alt="Screen Shot 2021-09-15 at 10 02 00 PM" src="https://user-images.githubusercontent.com/1396242/133552478-cc1a5989-170c-4e97-bf76-0a9dae44c364.png">

Things I have seen:
* This repo does not have `ext-tern.js` in it: https://github.com/ajaxorg/ace-builds/tree/master/src-noconflict
* The `ace-tern` repo does have `ext-tern.js` in it: https://github.com/sevin7676/Ace.Tern/tree/master/ace-builds/src-noconflict
* At the same time I found this project online that seems to use `ace-builds` and `tern` together successfully: https://github.com/tmone/incentive/blob/64a89587b002e9103f41a581165bb2ee214d0e79/NC.VS/Resource/lib/Ace_Tern/demo.html#L51

So, curious, should I be able to use `tern` with `ace-builds`?

cc @sevin7676

**Comment by JSmithOner** [comment]
@jparismorgan Sorry to answer so late. I've changed strategy but basically the way you rename seems odd and that's maybe why you get errors ( no problem with acorn ) I didn't get. My technic is hacky and usually you shouldn't place anything into the `node_modules`.Your best chance would be to dig deeper, but on my side I've made a couple of changes and done as so:

create a symlink for ace module in `angular.json`:

```
              "assets":[ {
                "glob": "**/*",
                "input": "./node_modules/ace-builds/src-noconflict/",
                "output": "/assets/js/ace"
              },{...}]
```

then add `tern-server.js` which is originally named `tern-ace.js` into a path `src/assets/js/ace` of your angular project.

install `tern` with `npm` or `yarn`

then in your component

```
{
...
  ace.config.set('basePath','/assets/js/ace');
  ace.config.loadModule( "ace/tern/server", ( module ) => {

      const ternServer = new module.TernServer();
  }
...
}
```
